test/parallel/test-util-inspect.js

```diff
@@ -1162,8 +1110,11 @@ util.inspect(process);
 
 // Setting custom inspect property to a non-function should do nothing.
```

Comment:
s/property/symbol/ ?

Reply:
But it is still a property. It is just accessed with a symbol.